demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java

```diff
@@ -51,12 +54,20 @@ public static void main(String args[]) {
 				
 				System.out.println(response.getCode());
 				System.out.println(response.getOptions());
-				System.out.println(response.getResponseText());
-				
-				System.out.println("\nADVANCED\n");
-				// access advanced API with access to more details through .advanced()
-				System.out.println(Utils.prettyPrint(response));
-				
+				if (args.length > 1) {
+					try (FileOutputStream out = new FileOutputStream(args[1])) {
+						out.write(response.getPayload());
```

Comment:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

using "try (Closable) {}" will do the `close()`.